Source/WebCore/platform/gstreamer/GStreamerQuirks.h

```diff
@@ -107,6 +127,13 @@ class GStreamerQuirksManager : public RefCounted<GStreamerQuirksManager> {
 
     bool shouldParseIncomingLibWebRTCBitStream() const;
 
+    bool needsBufferingPercentageCorrection() const;
+    // Returns name of the queried GstElement, or nullptr if no element was queried.
+    const char* queryBufferingPercentage(MediaPlayerPrivateGStreamer*, GRefPtr<GstQuery>&) const;
```

Comment:
oprah

Reply:
Poor Oprah... Don't tell her that in the end, [when she opens the ASCIILiteral wrapped gift](https://github.com/WebKit/WebKit/pull/29998/commits/11a45e5d030157ee664fe02ea9f770aad8dbd41f#diff-4376b854280205494dc9de0f08f985afd4d6584a7487c0a136adac2954c003cdR1340) there's going to be a `const char*` inside. So much wrapping paper wasted...

Should I also use ASCIILiteral in [MediaPlayerPrivateGStreamer::queryBufferingPercentage()](https://github.com/WebKit/WebKit/pull/29998/commits/11a45e5d030157ee664fe02ea9f770aad8dbd41f#diff-4376b854280205494dc9de0f08f985afd4d6584a7487c0a136adac2954c003cdR1336) too? Yeah, sure, let's do it. In the end, I'll still have to force a `static_cast<const char*>()` conversion when I need to [pass it to GST_TRACE_OBJECT()](https://github.com/WebKit/WebKit/pull/29998/commits/11a45e5d030157ee664fe02ea9f770aad8dbd41f#diff-4376b854280205494dc9de0f08f985afd4d6584a7487c0a136adac2954c003cdR1369), but who cares? Memory footprint and impedance caused by wrappers is cheap. 🤷

Reply:
no cast needed, there is a `charcacters()` method.

Reply:
Uh... Thank you! 🙏🙏🙏
I somehow missed it in the .h and stubbornly tried to force the usage of `constexpr operator const char*() const` through a static_cast. Though it worked, it felt a bit uncomfortable.